tagstudio/src/qt/ts_qt.py

```diff
@@ -164,16 +166,17 @@ class QtDriver(QObject):
     """A Qt GUI frontend driver for TagStudio."""
 
     SIGTERM = Signal()
+    settings_window = None
+    autosave_timer = QTimer()
```

Comment:
The instance of timer should be created in constructor instead of here. Moreover you're creating another instance later (line 519, so it creates second instance) which would overwrite this existing instance, so that's extra work for nothing.